WTWSMS/history/characters/roman.txt

```diff
@@ -5934,10 +5934,11 @@
 	450.1.1 = { death = yes }
 }
 1300792 = {
-	name = "Marcianus" #PLRE vol. 2 Iulius Agrius Tarrutentius Marcianus 20
+	name = "Tarrutenius Marcianus" #PLRE vol. 2 Iulius Agrius Tarrutentius Marcianus 20
 	# comes Orientis soon after 437
+	dynasty = 8648
```

Comment:
Octavianus?

Reply:
Yes, he is descendant of Augustus through his father.

Reply:
What father? There is no father defined here.

Reply:
Tarrutenius Maximilianus is his father. I connected them based on PLRE

Reply:
Tarrutenius Maximilianus is his father. I connected them based on PLRE.

Reply:
There is no father entry.

Reply:
In articles Iulius Agrius Tarrutenius Marcianus 20 and Tarrutenius Maximilianus 3 it is said about possible connection between them.

Reply:
@loup99 is this resolved?

Reply:
No.

Reply:
Now this is resolved, i just thought that i already added father.